com.unity.render-pipelines.high-definition/Editor/RenderPipeline/Settings/QualitySettingsPanel.cs

```diff
@@ -97,6 +103,17 @@ public void OnGUI(string searchContext)
                     Editor.CreateCachedEditor(asset.asset, typeof(HDRenderPipelineEditor), ref m_Cached);
                     ((HDRenderPipelineEditor)m_Cached).largeLabelWidth = false;
                     m_Cached.OnInspectorGUI();
+
+                    if (Highlighter.active)
+                    {
+                        if (Highlighter.activeVisible)
+                        {
+                            if (Time.realtimeSinceStartup - m_HighlightStart > s_HighlightDuration)
+                                Highlighter.Stop();
+                        }
+                        else
+                            m_HighlightStart = Time.realtimeSinceStartup;
+                    }
```

Comment:
I wonder if we could extract this functionality somewhere on CoreEditorUtils, It might be interesting to have it easily for the future.